### PR TITLE
[observability] Add gRPC client call and additional functional metrics

### LINKED ISSUE #5706: Logs: Log entries related content initialiser should be findable when searching for a workspace
When searching for the log entries that explain why content init failed, they can be not found when searching for workspace name or ID.

### PR DESCRIPTION
**Pull request by csweichel**
## Description
Adds client call metrics to gRPC calls form TS and Go, and makes the server -> image-builder connection use the default gRPC unary options. The lack of the latter probably was the cause of the recent image-builder related outage.

Also adds cluster registration metrics to ws-manager-bridge, and image-build counts to image-builder-mk3

Thanks @JanKoehnlein for figuring out the nitty gritty of the grpc-js interceptor business.

## Related Issue(s)
Fixes #5706, #5707

## How to test
1. start a workspace to produce some data all the way 'round
2. look at the metric endpoints of `server`, `image-builder-mk3`, `ws-manager-bridge`, `ws-manager`, `ws-proxy`, `ws-scheduler`. You should see `grpc_client_calls_total` metrics on all those services.

## Release Notes

```release-note
[server] Add gRPC client call metrics
[ws-manager-bridge] Add gRPC client call and cluster registration metrics
[image-builder-mk3] Add image-build metrics
```

### CODE REVIEW

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please ask for approval from **csweichel** after the PR has been reviewed.

Associated issue: *#5706*

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/gitpod-io/gitpod/blob/main/OWNERS)**
- **[components/common-go/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/common-go/OWNERS)**
- **[components/content-service-api/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/content-service-api/OWNERS)**
- **[components/ee/ws-scheduler/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/ee/ws-scheduler/OWNERS)**
- **[components/gitpod-protocol/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/gitpod-protocol/OWNERS)**
- **[components/image-builder-api/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/image-builder-api/OWNERS)**
- **[components/registry-facade/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/registry-facade/OWNERS)**
- **[components/server/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/server/OWNERS)**
- **[components/ws-manager/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/ws-manager/OWNERS)**
- **[components/ws-manager-api/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/ws-manager-api/OWNERS)**
- **[components/ws-manager-bridge/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/ws-manager-bridge/OWNERS)**
- **[components/ws-proxy/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/ws-proxy/OWNERS)**
- **[dev/gpctl/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/dev/gpctl/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: components/image-builder-mk3/pkg/orchestrator/metrics.go [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+	}
+}
+
+func (m *metrics) BuildDone(success bool) {
+	m.imageBuildsDoneTotal.WithLabelValues(strconv.FormatBool(success)).Inc()
+}
+
+func (m *metrics) BuildStarted() {
+	m.imageBuildsStartedTotal.Inc()
+}
```
Comment (ArthurSens) [inline components/image-builder-mk3/pkg/orchestrator/metrics.go:57]: Really liked the metrics chosen here!

We can:
1. Calculate the percentage of image builds that have started but are not finishing
2. Calculate the percentage of image builds that finishes successfully

File: components/gitpod-protocol/src/messaging/client-call-metrics.ts [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+				grpc_method: labels.method,
+				grpc_type: labels.type,
+				grpc_code: labels.code,
+			});
+		} else {
+			this.counter.inc({
+				grpc_service: labels.service,
+				grpc_method: labels.method,
+				grpc_type: labels.type
+			});
```
Comment (ArthurSens) [inline components/gitpod-protocol/src/messaging/client-call-metrics.ts:49]: Does it really work if you try to increment without passing one of the labels? (`grpc_code` in this case)

File: components/image-builder-mk3/pkg/orchestrator/metrics.go (hunk 1 above)
Comment (ArthurSens) [inline components/image-builder-mk3/pkg/orchestrator/metrics.go:57]: Really liked the metrics chosen here!

We can:
1. Calculate the percentage of image builds that have started but are not finishing
2. Calculate the percentage of image builds that finishes successfully

File: components/gitpod-protocol/src/messaging/client-call-metrics.ts [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+		}
```
Comment (ArthurSens) [inline components/gitpod-protocol/src/messaging/client-call-metrics.ts:50]: Does it really work if you try to increment without passing one of the labels? (`grpc_code` in this case)

File: components/registry-facade/cmd/run.go [hunk 4]
```diff
@@ -43,6 +45,34 @@ var runCmd = &cobra.Command{
 			log.WithError(err).WithField("filename", configPath).Fatal("cannot load config")
 		}
 
+		promreg := prometheus.NewRegistry()
+		gpreg := prometheus.WrapRegistererWithPrefix("gitpod_registry_facade_", promreg)
+		rtt, err := registry.NewMeasuringRegistryRoundTripper(newDefaultTransport(), prometheus.WrapRegistererWithPrefix("downstream_", gpreg))
+		if err != nil {
+			log.WithError(err).Fatal("cannot registry metrics")
```
Comment (ArthurSens) [inline components/registry-facade/cmd/run.go:52]: small nit
```suggestion
			log.WithError(err).Fatal("cannot register metrics")
```

File: components/registry-facade/cmd/run.go [hunk 5]
```diff
@@ -43,6 +45,34 @@ var runCmd = &cobra.Command{
...
+		}
+		if cfg.PrometheusAddr != "" {
+			promreg.MustRegister(
+				collectors.NewGoCollector(),
+				collectors.NewProcessCollector(collectors.ProcessCollectorOpts{}),
+				common_grpc.ClientMetrics(),
+			)
+
+			handler := http.NewServeMux()
+			handler.Handle("/metrics", promhttp.HandlerFor(promreg, promhttp.HandlerOpts{}))
```
Comment (ArthurSens) [inline components/registry-facade/cmd/run.go:62]: I see that you create a new registry called `gpreg` to add the "gitpod_registry_facade" prefix, but then you use the previous registry, `promreg`, to serve the metrics endpoint.

Is that intended?

Comment (ArthurSens) [reply components/registry-facade/cmd/run.go:62]: I know that you just moved code to another location, it's just that it looks awkward 😅

File: components/ws-manager-bridge/src/prometheus-metrics-exporter.ts [hunk 6]
```diff
@@ -26,6 +29,16 @@ export class PrometheusMetricsExporter {
             labelNames: ['region'],
             buckets: prom.exponentialBuckets(2, 2, 10),
         });
+        this.clusterScore = new prom.Gauge({
+            name: 'gitpod_ws_manager_bridge_cluster_score',
+            help: 'Score of the individual registered workspace cluster',
+            labelNames: ["cluster"]
```
Comment (ArthurSens) [inline components/ws-manager-bridge/src/prometheus-metrics-exporter.ts:35]: There is no need to add the cluster label. Prometheus already adds this label to all metrics when it remote-writes to a Long Term Storage.

If we keep it, we'll just have duplicated labels

Comment (ArthurSens) [reply components/ws-manager-bridge/src/prometheus-metrics-exporter.ts:35]: If you really want to keep this label. I'd ask to at least change the label name... On monitoring-satellite configuration we've already made `cluster` sort of a "reserved" label that is applied to every metric 😬

**Review by ArthurSens — COMMENTED** [review]
My first walk through the code...

I really liked the decision of using go-grpc-middleware to expose client-side gRPC metrics! When you announced this in the all-hands I thought you were going to implement this in common-go alone 😅.

File: components/gitpod-protocol/src/messaging/client-call-metrics.ts (hunk 3 above)
Comment (ArthurSens) [reply components/gitpod-protocol/src/messaging/client-call-metrics.ts:50]: I'd expect an error from the library

Comment (JanKoehnlein) [reply components/gitpod-protocol/src/messaging/client-call-metrics.ts:50]: I'm having a look on that. In particular we need 4 different counters to be on par with the Go middleware. 
I can also do that in a separate PR, so don't feel blocked to merge this. The call without `grpc_code` is a dead branch ATM anyway.

Comment (JanKoehnlein) [reply components/gitpod-protocol/src/messaging/client-call-metrics.ts]: Added a fix

File: components/registry-facade/cmd/run.go (hunk 5 above)
Comment (csweichel) [reply components/registry-facade/cmd/run.go:62]: `gpreg` is used in two places and just wraps the original registerer. I don't think there's much to be changed here.

File: components/registry-facade/cmd/run.go (hunk 4 above)
Comment (csweichel) [reply components/registry-facade/cmd/run.go:52]: :+1:

File: components/image-builder-mk3/pkg/orchestrator/metrics.go (hunk 1 above)
Comment (csweichel) [reply components/image-builder-mk3/pkg/orchestrator/metrics.go:57]: :+1:

File: components/ws-manager-bridge/src/prometheus-metrics-exporter.ts (hunk 6 above)
Comment (csweichel) [reply components/ws-manager-bridge/src/prometheus-metrics-exporter.ts:35]: This is not the same cluster as the one this metric originates from. Instead, it's the registerd workspace cluster for which we're reporting the score.
I'll change this to `workspace_cluster`.

**Comment by csweichel** [comment]
I've addressed the comments. @ArthurSens and @geropl please have another look.

**Comment by roboquat** [comment]
LGTM label has been added.  <details>Git tree hash: 6ff787bc06800936c3677f0c554ed15998326d7b</details>

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/gitpod-io/gitpod/pull/5726#issuecomment-921612138" title="LGTM">JanKoehnlein</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please ask for approval from **csweichel** after the PR has been reviewed.

Associated issue: *#5706*

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

<details open>
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/gitpod-io/gitpod/blob/main/OWNERS)~~ [JanKoehnlein]
- **[components/common-go/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/common-go/OWNERS)**
- **[components/content-service-api/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/content-service-api/OWNERS)**
- **[components/ee/ws-scheduler/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/ee/ws-scheduler/OWNERS)**
- ~~[components/gitpod-protocol/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/gitpod-protocol/OWNERS)~~ [JanKoehnlein]
- **[components/image-builder-api/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/image-builder-api/OWNERS)**
- **[components/registry-facade/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/registry-facade/OWNERS)**
- ~~[components/server/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/server/OWNERS)~~ [JanKoehnlein]
- **[components/ws-manager/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/ws-manager/OWNERS)**
- **[components/ws-manager-api/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/ws-manager-api/OWNERS)**
- ~~[components/ws-manager-bridge/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/ws-manager-bridge/OWNERS)~~ [JanKoehnlein]
- **[components/ws-proxy/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/ws-proxy/OWNERS)**
- **[dev/gpctl/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/dev/gpctl/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by JanKoehnlein** [comment]
/approve cancel

Looking at our logs, there seems to be a severe problem with ws-manager in core-dev workspaces with this change
https://console.cloud.google.com/logs/query;query=%0Aseverity%3DERROR%0Aresource.labels.namespace_name%3D%22staging-cw-fix-5706-5707%22%0Aresource.labels.pod_name%3D%22ws-daemon-xcmp6%22;timeRange=PT1H;summaryFields=labels%252F%2522compute.googleapis.com%252Fresource_name%2522:false:32:beginning;cursorTimestamp=2021-09-17T09:08:54Z?project=gitpod-core-dev

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please ask for approval from **csweichel** after the PR has been reviewed.

Associated issue: *#5706*

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/gitpod-io/gitpod/blob/main/OWNERS)**
- **[components/common-go/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/common-go/OWNERS)**
- **[components/content-service-api/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/content-service-api/OWNERS)**
- **[components/ee/ws-scheduler/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/ee/ws-scheduler/OWNERS)**
- **[components/gitpod-protocol/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/gitpod-protocol/OWNERS)**
- **[components/image-builder-api/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/image-builder-api/OWNERS)**
- **[components/registry-facade/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/registry-facade/OWNERS)**
- **[components/server/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/server/OWNERS)**
- **[components/ws-manager/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/ws-manager/OWNERS)**
- **[components/ws-manager-api/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/ws-manager-api/OWNERS)**
- **[components/ws-manager-bridge/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/ws-manager-bridge/OWNERS)**
- **[components/ws-proxy/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/ws-proxy/OWNERS)**
- **[dev/gpctl/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/dev/gpctl/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by aledbf** [comment]
> Looking at our logs, there seems to be a severe problem with ws-manager in core-dev workspaces with this change

The error you see in the logs was fixed here https://github.com/gitpod-io/gitpod/pull/5696, i.e., is not related to this PR

**Comment by JanKoehnlein** [comment]
Thanks for the clarification @aledbf. I just wanted to make sure we don't commit anything that sets our logging on fire

/approve

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/gitpod-io/gitpod/pull/5726#issuecomment-921717007" title="Approved">JanKoehnlein</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please ask for approval from **csweichel** after the PR has been reviewed.

Associated issue: *#5706*

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

<details open>
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/gitpod-io/gitpod/blob/main/OWNERS)~~ [JanKoehnlein]
- **[components/common-go/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/common-go/OWNERS)**
- **[components/content-service-api/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/content-service-api/OWNERS)**
- **[components/ee/ws-scheduler/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/ee/ws-scheduler/OWNERS)**
- ~~[components/gitpod-protocol/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/gitpod-protocol/OWNERS)~~ [JanKoehnlein]
- **[components/image-builder-api/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/image-builder-api/OWNERS)**
- **[components/registry-facade/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/registry-facade/OWNERS)**
- ~~[components/server/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/server/OWNERS)~~ [JanKoehnlein]
- **[components/ws-manager/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/ws-manager/OWNERS)**
- **[components/ws-manager-api/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/ws-manager-api/OWNERS)**
- ~~[components/ws-manager-bridge/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/ws-manager-bridge/OWNERS)~~ [JanKoehnlein]
- **[components/ws-proxy/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/ws-proxy/OWNERS)**
- **[dev/gpctl/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/dev/gpctl/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/gitpod-io/gitpod/pull/5726#issuecomment-921718728" title="Approved">aledbf</a>*, *<a href="https://github.com/gitpod-io/gitpod/pull/5726#issuecomment-921717007" title="Approved">JanKoehnlein</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please ask for approval from **csweichel** after the PR has been reviewed.

Associated issue: *#5706*

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

<details open>
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/gitpod-io/gitpod/blob/main/OWNERS)~~ [JanKoehnlein,aledbf]
- ~~[components/common-go/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/common-go/OWNERS)~~ [aledbf]
- **[components/content-service-api/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/content-service-api/OWNERS)**
- ~~[components/ee/ws-scheduler/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/ee/ws-scheduler/OWNERS)~~ [aledbf]
- ~~[components/gitpod-protocol/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/gitpod-protocol/OWNERS)~~ [JanKoehnlein]
- **[components/image-builder-api/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/image-builder-api/OWNERS)**
- ~~[components/registry-facade/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/registry-facade/OWNERS)~~ [aledbf]
- ~~[components/server/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/server/OWNERS)~~ [JanKoehnlein]
- ~~[components/ws-manager/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/ws-manager/OWNERS)~~ [aledbf]
- ~~[components/ws-manager-api/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/ws-manager-api/OWNERS)~~ [aledbf]
- ~~[components/ws-manager-bridge/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/ws-manager-bridge/OWNERS)~~ [JanKoehnlein]
- ~~[components/ws-proxy/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/ws-proxy/OWNERS)~~ [aledbf]
- ~~[dev/gpctl/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/dev/gpctl/OWNERS)~~ [aledbf]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

Approval requirements bypassed by manually added approval.

This pull-request has been approved by: *<a href="https://github.com/gitpod-io/gitpod/pull/5726#issuecomment-921718728" title="Approved">aledbf</a>*, *<a href="https://github.com/gitpod-io/gitpod/pull/5726#issuecomment-921717007" title="Approved">JanKoehnlein</a>*

Associated issue: *#5706*

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/gitpod-io/gitpod/blob/main/OWNERS)~~ [JanKoehnlein,aledbf]
- ~~[components/common-go/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/common-go/OWNERS)~~ [aledbf]
- **[components/content-service-api/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/content-service-api/OWNERS)**
- ~~[components/ee/ws-scheduler/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/ee/ws-scheduler/OWNERS)~~ [aledbf]
- ~~[components/gitpod-protocol/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/gitpod-protocol/OWNERS)~~ [JanKoehnlein]
- **[components/image-builder-api/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/image-builder-api/OWNERS)**
- ~~[components/registry-facade/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/registry-facade/OWNERS)~~ [aledbf]
- ~~[components/server/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/server/OWNERS)~~ [JanKoehnlein]
- ~~[components/ws-manager/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/ws-manager/OWNERS)~~ [aledbf]
- ~~[components/ws-manager-api/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/ws-manager-api/OWNERS)~~ [aledbf]
- ~~[components/ws-manager-bridge/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/ws-manager-bridge/OWNERS)~~ [JanKoehnlein]
- ~~[components/ws-proxy/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/ws-proxy/OWNERS)~~ [aledbf]
- ~~[dev/gpctl/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/dev/gpctl/OWNERS)~~ [aledbf]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: components/ws-manager-bridge/src/prometheus-metrics-exporter.ts (hunk 6 above)
Comment (ArthurSens) [reply components/ws-manager-bridge/src/prometheus-metrics-exporter.ts]: Hmmm thanks for the clarification! For some reason I thought ws-manager-bridge was running in the workspace clusters 🤦‍♂️.

This means that my implementation in https://github.com/gitpod-io/gitpod/pull/5557 was completely wrong

**Pull request merged**